docs/src/ice_cream_straight_sides.control

```diff
@@ -0,0 +1,57 @@
+\begin{CONTROL_INPUT}
+
+   \begin{RUN_PARAMETERS}
```

Comment:
Maybe it's useful to generate these files automatically from the text in `hohq_literate.jl` such that not every change has to be adapted three times.

Reply:
That's a good idea.